code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -1,6 +1,10 @@
 //Deployable turrets. They can be either automated, manually fired, or installed with a pAI.
 //They are built in stages, and only engineers have access to them.
 
+#define SENTRY_ALERT_AMMO			0
+#define SENTRY_ALERT_HOSTILE		1
+#define SENTRY_ALERT_FALLEN			2
```

Comment:
Put these in _Defines. Using defines in a place like this might mean theyre defined after code that uses them wants to use them. (And these seem like defines you might want to know outside of this file)